EIPS/eip-7503.md

```diff
@@ -0,0 +1,111 @@
+functionality to Ethereum, so that people can re-mint Ethers that they have purposefully burnt.
+This will make an anonymity pool of the size of the entire Ethereum blockchain without much hassle.
+
+## Specification
+
+In Elliptic-Curve based digital signatures, normally there is a secret scalar $s$, from which 
+a public-key is deriven (By multiplying the generator point with the scalar: $s \times G$)
+
+A public-key is spendable if and only if its corresponding private-key $s$ exists. We can
+generate unspendable public-keys by generating random points on the curve. But how can other 
+people be sure that a point is indeed random and not the result of calculating $s \times G$?
+We can generate points that are very unlikely to be spendable by using fancy-looking patterns 
+in their $x$ coordinate. E.g: In case of Secp256k1 we can pick $x=123456789$ and calculate 
+$y$ by putting it in the curve equation:
+
+$y^2=x^3+7$
+
+Because of the discrete logarithm problem, it's practically impossible to calculate $s$ where 
+$s \times G$ is equal with our point.
+
+Let's say $R$ is an unspendable public-key (Let's call it the reference unspendable public-key). 
+People can publicly burn their tokens by sending them to $R$, after doing so, others can indeed 
+conclude that they have burnt their tokens, because they can all see the transactions and they 
+know that $R$ is unspendable.
+
+***We can derive infinitely many provably-unspendable public-keys from a reference unspendable public-key***
+
+Let's pick a random secret value $t$ and calculate a new point $D=R + t \times G$. We can prove 
+that $D$ is also unspendable, because $log_G(D)=log_G(R + t \times G)=log_G(R) + t$, and since 
+we can't calculate $log_G(R)$, the public-key $D$ is also unspendable.
+
+Obviously, $D$ is a completely new point that does not seem unspendable. We can convince others 
+that $D$ is unspendable by revealing $t$, because then people can verify that $D$ is the result 
+of adding some other point to $R$.
+
+***Using the help of Zero-Knowledge proofs, we can hide the value of $t$***
+
+We just need to prove that we know a secret value $t$ where $R + t \times G == D$. We can go even 
+further. We can prove that some EOA-to-EOA transaction has happened in the history of 
+blockchain, which has burnt some amount of ETH. By revealing this to the Ethereum blockchain and 
+providing something like a nullifier (E.g. hash of the secret value $t$ so that double minting of 
+same burnt tokens are not possible), we can add a new ***minting*** functionality for ETH so that
+people can migrate they secretly burnt tokens to a completely new address, without any trace on 
+the blockchain. This will make an anonymity pool of the size of the entire Ethereum blockchain 
+without much hassle.
+
+## Rationale
+
+Cryptocurrency mixers like TornadoCash can successfully obfuscate Ethereum transactions, but it's
```

Comment:
TC to this EIP comparison looks apples to oranges as this EIP doesn't preserve assets/tokens , unless natively supported by the protocol (for eth) or by smart contracts (for tokens).

Reply:
Correct, but if we limit ourselves to ETH transfers, this EIP and TC are comparable imo.